### PR TITLE
pointing schema: rename (ra, dec)

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
# Description
Rename property (ra, dec) as present in localization schema as well.

# Related Issue(s)
https://github.com/nasa-gcn/gcn-schema/issues/145

### CODE REVIEW

**Review by lpsinger — COMMENTED** [review]
The name change is fine, but I think that the pointing schema is underspecified and ill-defined. What frame is this in? ICRS? GCRS? HCRS?

**Comment by Vidushi-GitHub** [comment]
> The name change is fine, but I think that the pointing schema is underspecified and ill-defined. What frame is this in? ICRS? GCRS? HCRS?

Thank you, I was also in similar thought, while comparing with Localization Schema.

File: gcn/notices/core/Pointing.schema.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
   "$comment": "Providers should $ref this schema directly but Attitudes.schema.json instead.",
   "type": "object",
   "properties": {
-    "ra": {
+    "ra_pointing": {
       "type": "number",
       "description": "RA pointing of the instrument Z axis"
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Pointing.schema.json:11]:
```suggestion
      "description": "RA pointing of the instrument Z axis. ICRS declination [deg], utilizes the J2000 epoch and an equatorial coordinate system."
```

File: gcn/notices/core/Pointing.schema.json [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
     },
-    "dec": {
+    "dec_pointing": {
       "type": "number",
       "description": "Dec pointing of the instrument Z axis"
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Pointing.schema.json:15]:
```suggestion
      "description": "Dec pointing of the instrument Z axis.  ICRS declination [deg], utilizes the J2000 epoch and an equatorial coordinate system."
```

**Review by modot — APPROVED** [review]

**Review by lpsinger — APPROVED** [review]

**Comment by lpsinger** [comment]
@Vidushi-GitHub, next time please remember to create branches for PRs on your own fork.

**Pull request merged**